contentgrid-automations-rest/src/main/java/com/contentgrid/spring/automations/AutomationLinkRelations.java

```diff
@@ -14,5 +14,6 @@ public class AutomationLinkRelations {
     public static final LinkRelation ANNOTATION = HalLinkRelation.curied(CURIE, "annotation");
     public static final String ANNOTATION_STRING = CURIE+":annotation";
     public static final LinkRelation TARGET_ENTITY = HalLinkRelation.curied(CURIE, "target-entity");
+    public static final LinkRelation REGISTRATIONS = HalLinkRelation.curied(CURIE, "registrations");
```

Comment:
Wouldn't this just be `automations:automations`? Because it's not like you can register at that endpoint.

Reply:
It's right that you cannot add registrations here, but you can list the registrations that already there.